packages/calypso-products/src/plans-list.tsx

```diff
@@ -734,10 +748,16 @@ const getJetpackBusinessDetails = (): IncompleteJetpackPlan => ( {
 			PLAN_JETPACK_PERSONAL_MONTHLY,
 		].includes( plan ),
 	getDescription: () =>
-		i18n.translate(
-			'{{strong}}Best for organizations:{{/strong}} The most powerful WordPress sites.',
-			plansDescriptionHeadingComponent
-		),
+		isEnabled( 'themes/premium' )
+			? i18n.translate(
+					'{{strong}}Best for organizations:{{/strong}} The most powerful WordPress sites.',
+					plansDescriptionHeadingComponent
+			  )
+			: i18n.translate(
+					'{{strong}}Best for organizations:{{/strong}} The most powerful WordPress sites: real-time backups ' +
```

Comment:
:information_source: String reuse speeds up translation and improves consistency. The following string might make a good alternative and has already been translated **1** times:
`translate( '{{strong}}Best for organizations:{{/strong}} The most powerful WordPress sites: real-time backups and unlimited premium themes.' )` *ES Score: 10*
See 2 additional suggestions in the [PR translation status page](https://translate.wordpress.com/localci/status/automattic/wp-calypso/add/themes-premium-ref-to-start-plans)